docs/source/index.mdx

```diff
@@ -16,6 +16,8 @@ limitations under the License.
 
 # 🤗 Optimum TPU
 
+TEST
```

Comment:
I guess you did not want this here, did you?

Reply:
It's just a test to trigger the PR doc build. It is triggered only when the doc is modified. I'll remove it once the preview is generated.